packages/cf-builder-card/src/cardReducer.js

```diff
@@ -1,10 +1,12 @@
-const ActionTypes = require('./CardActionTypes');
+// @flow
 
-const initialState = {};
+import type {CardAction, CardState} from './CardTypes';
```

Comment:
That would be fine and dandy except the TC39 is currently trying to figure out what to do with modules because they can't change some semantics about them because Babel users are already using them.

So no, there is no "standardized" module spec. There's just a syntax which is now causing problems and we might need to break things for everyone.

But if I'm wrong, then I'd love to hear the context you have that I don't.